src/modules/Dashboard/hooks/useDecryptedActivityData.ts

```diff
@@ -85,6 +88,17 @@ export const useDecryptedActivityData = (
       }
     }
 
+    checkIfShouldLogging() &&
+      logDataInDebugMode({
```

Comment:
it seems `logDataInDebugMode` already has a check for `checkIfShouldLogging`?
`export const logDataInDebugMode = (data: unknown) => checkIfShouldLogging() && console.log(data);`

Reply:
Yes, that's right! I'll get rid of it.